src/domain/application/components/ApplicationSendDialog.tsx

```diff
@@ -142,24 +174,40 @@ const ApplicationSendDialog: React.FC<Props> = ({ type, isOpen, isLoading, onClo
                       name="paperDecisionReceiver.city"
                       label={t('hakemus:sendDialog:city')}
                       required={orderPaperDecisionChecked}
+                      disabled={isSending}
                     />
                   </GridItem>
                 </Grid>
               </Box>
             )}
+            {isError && (
```

Comment:
Same here as above that mutations `isError` could be used.

Reply:
Changed it.